examples/s3-object-lambda/index.ts

```diff
@@ -1,7 +1,19 @@
+import * as pulumi from '@pulumi/pulumi';
+import * as pulumicdk from '@pulumi/cdk';
+import type { AppOutputs } from '@pulumi/cdk';
 import { S3ObjectLambdaStack } from './src/s3-object-lambda-stack';
 
-const s = new S3ObjectLambdaStack('stack');
-export const exampleBucketArn = s.exampleBucketArn;
-export const objectLambdaArn = s.objectLambdaArn;
-export const objectLambdaAccessPointArn = s.objectLambdaAccessPointArn;
-export const objectLambdaAccessPointUrl = s.objectLambdaAccessPointUrl;
+const app = new pulumicdk.App('app', (scope: pulumicdk.App): AppOutputs => {
+    const s = new S3ObjectLambdaStack(scope, 'stack');
+    return {
```

Comment:
Ah that's a bit unfortunate that we now need to duplicate the outputs in order to export them :/

But I guess we could just return all the stacks here and to the unwrapping below, right?

i.e.
```
return {
    lambdaStack: s,
}

// ...
// root level
export const exampleBucketArn = app.outputs['lambdaStack'].exampleBucketArn;
```

Reply:
Yep that's true. I experimented with a couple of things, but since we have to create resources within that callback function the best I could  come up with was this untyped `app.outputs` option.